applications/GeoMechanicsApplication/custom_utilities/hydraulic_discharge.cpp

```diff
@@ -0,0 +1,50 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Anne van de Graaf
+//
+
+#include "custom_utilities/hydraulic_discharge.h"
```

Comment:
See my other comment for the header includes, I think a complete list would be:
```suggestion
#include "hydraulic_discharge.h"
#include "custom_utilities/node_utilities.h"
#include "geo_mechanics_application_variables.h"
#include "includes/ublas_interface.h"
```

Reply:
done